Integrations/integration-ServiceNow.yml

```diff
@@ -96,11 +95,17 @@ script:
         }
     };
 
-    var getBody = function() {
+    var getBody = function(templateName) {
         var body = {};
+        if (templateName !== undefined) var tmpRes = SNGetTemplate(templateName);
```

Comment:
don't use such if ... declaration statements, this is confusing as coders will search declarations in the left side
write it in the following manner
var tmpRes;
if (...) {
tmpRes = ...
}